lib/PAUSE/package.pm

```diff
@@ -83,12 +83,27 @@ sub give_regdowner_perms {
   my $self = shift;
   my $dbh = $self->connect;
   my $package = $self->{PACKAGE};
+  my $main_package = $self->{MAIN_PACKAGE};
   local($dbh->{RaiseError}) = 0;
   # warn "Going to execute [SELECT userid FROM mods WHERE modid = '$package']";
-  my ($sth_mods) = $dbh->selectrow_hashref("SELECT userid
+  my $sql = "SELECT userid
                                 FROM   mods
-                                WHERE  modid = ?", undef, $package);
-  if ($sth_mods) { # make sure we regard the owner as the owner
+                                WHERE  modid = ?
+                                ";
+  my @args = ($package);
+  unless(lc($main_package) eq lc($package)) {
+    $sql .= q{
+    UNION
+    SELECT userid
+    FROM   perms
+    WHERE  LOWER(package) = LOWER(?)
```

Comment:
Not a big deal, but: this is MySQL, so this `=` is already case-insensitive, right?  If we're going to use an explicit LOWER on both sides (which seems fine) we should probably also add it to the `WHERE modid = ?` above, too.